contributing/code/standards.rst

```diff
@@ -167,6 +167,23 @@ Service Naming Conventions
 
 * A group name uses the underscore notation.
 
+.. _contributing-code-standards-comparisons:
+
+Comparisons
+-----------
+
+Always use `identical comparison`_ unless you need type juggling::
```

Comment:
No problem at all, I'll try to update this PR and #5402 ASAP based in your suggestions.
Thank you.